### PR TITLE
fix: migrate empty states to meteor components

### PR DESCRIPTION
**Pull request by Weltraumakustik**
### 1. Why is this change necessary?

Several Administration list views showed incomplete, missing, or visually inconsistent empty states after migrating toward Meteor components. Some affected views still rendered table headers without useful empty-state content, while others had incorrect alignment, spacing, or toolbar layout.

The integrations list also regressed from the intended Meteor data table implementation back to a legacy listing/card mix, which broke table layout, pagination, page-size controls, and discoverability for larger integration lists.

### 2. What does this change do, exactly?

- Migrates and polishes affected Administration empty states to use Meteor empty-state patterns consistently.
- Fixes empty-state placement, alignment, card spacing, and toolbar layout in affected modules.
- Restores the integrations list to the Meteor `mt-data-table`.
- Keeps integrations pagination and page-size controls usable for larger lists, including after selecting `50` items per page.
- Enables Meteor search and permission filters for integration lists with more than 25 items.
- Updates focused Administration unit tests for the changed empty states, toolbars, and integrations table behavior.
- Documents developer-facing changes in:
  - `RELEASE_INFO-6.7.md`
  - `UPGRADE-6.8.md`
  
There for sure are **still edge-cases** that require more deep fixes. In some cases an upgrade to meteor components would already do the trick but this needs to be coordinated with the meteor team and other areas. 

### 3. Describe each step to reproduce the issue or behaviour.

1. Open the Administration.
2. Visit affected list/detail views with no results or empty related data, for example:
   - property values
   - customer orders
   - customer addresses
   - reviews
   - countries
   - mail templates
   - users and roles permissions
3. Observe missing, misaligned, or incorrectly placed empty states and broken toolbar/table layouts.
4. Open `#/sw/integration/index`.
5. Create or load more than 25 integrations.
6. Observe that the integration table should expose search, filters, pagination, and page-size controls without hiding the footer after changing the page size.

### 4. Please link to the relevant issues (if any).

- relates #9495

### 5. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have updated developer-facing release notes if this change is **relevant** for external developers:
  - Added an entry to `RELEASE_INFO-6.7.md` under Administration for the integration list Meteor data table migration.
  - Added `UPGRADE-6.8.md` sections for the breaking Twig block changes and integration list migration guidance.
  - See the [Documenting a Release Process](https://github.com/shopware/shopware/blob/trunk/delivery-process/documenting-a-release.md) for details.
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfilled them

### Screenshots
notifications-bell

cross-selling

updater

orders

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/app/component/base/sw-property-search/sw-property-search.html.twig [hunk 1]
```diff
@@ -220,11 +220,7 @@
             :icon="$route.meta.$module.icon"
             :headline="$t('sw-property-search.noPropertiesFound')"
             :description="$t('sw-property-search.noPropertiesFoundDescription')"
-        >
-            <!-- @deprecated tag:v6.8.0 - block will be removed -->
-            {% block sw_property_search_empty_image %}
-            {% endblock %}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/app/component/base/sw-property-search/sw-property-search.html.twig:226]: Don't remove blocks in minor versions

File: src/Administration/Resources/app/administration/src/app/component/utils/sw-notification-center/index.js [hunk 2]
```diff
@@ -71,6 +97,9 @@ export default {
         onCloseDeleteModal() {
             this.showDeleteModal = false;
         },
+        onEmptyStateBellClick() {
+            this.emptyStateBellAnimationVariant = this.emptyStateBellAnimationVariant === 'a' ? 'b' : 'a';
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/app/component/utils/sw-notification-center/index.js]: What does a or b mean in this context? If it works, "active/inactive" would be better

File: src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-layout-assignment-modal/index.ts [hunk 3]
```diff
@@ -143,10 +143,6 @@ export default Shopware.Component.wrapComponentConfig({
             return this.page.type === 'product_detail';
         },
 
-        assetFilter() {
-            return Shopware.Filter.getByName('asset');
-        },
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-layout-assignment-modal/index.ts:148]: Please deprecate instead

File: src/Administration/Resources/app/administration/src/module/sw-customer/view/sw-customer-detail-order/sw-customer-detail-order.html.twig [hunk 4]
```diff
@@ -92,14 +92,19 @@
...
             icon="solid-shopping-bag"
             :headline="emptyTitle"
-            :description="$t('sw-customer.detailOrder.emptySubline')"
+            :description="emptyDescription"
+            :role="term ? 'status' : undefined"
+            :aria-live="term ? 'polite' : undefined"
+            :aria-atomic="term ? 'true' : undefined"
         >
-            <template #button>
-                {% block sw_customer_detail_order_card_grid_empty_state_action %}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-customer/view/sw-customer-detail-order/sw-customer-detail-order.html.twig:98]: Please deprecate instead

File: src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-ratings/sw-extension-ratings-card/sw-extension-ratings-card.html.twig [hunk 5]
```diff
@@ -38,9 +38,11 @@
 
     {% block sw_extension_ratings_card_empty_state %}
     <template v-else>
-        {% block sw_extension_ratings_card_empty_state_content %}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-ratings/sw-extension-ratings-card/sw-extension-ratings-card.html.twig:41]: Please deprecate instead

File: src/Administration/Resources/app/administration/src/module/sw-flow/view/listing/sw-flow-list-flow-templates/sw-flow-list-flow-templates.html.twig [hunk 6]
```diff
@@ -52,11 +52,11 @@
             </sw-entity-listing>
 
             <mt-empty-state
-                v-else
+                v-if="!total && !isLoading"
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-flow/view/listing/sw-flow-list-flow-templates/sw-flow-list-flow-templates.html.twig]: Is this really a case that this will be shown when either case is happening? I'd assume `v-else` would be fine here, and `isLoading && total` would be correct above.

If it is necessary: maybe `v-else-if=...` is an option here, so it won't be checked when the entity listing is rendered in the first place

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/index.js [hunk 7]
```diff
@@ -69,14 +86,94 @@ export default {
...
+                    property: 'permissions',
+                    label: this.$t('sw-integration.list.permissions'),
+                    renderer: 'text',
+                    position: 1,
                 },
+            ];
+        },
+
+        hasIntegrations() {
+            return this.integrations && this.integrations.length > 0;
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/index.js]:
```suggestion
            return this.integrations?.length > 0;
```

You can make this shorter with the `?`, meaning, that if the prior part (here "integrations") is rendered falsy, it wouldn't try to continue. So it's the short form / more modern of what you wrote.

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig [hunk 8]
```diff
@@ -209,137 +205,159 @@
...
-                    :columns="integrationColumns"
-                    :repository="integrationRepository"
-                    :full-page="false"
-                    :compact-mode="false"
-                    :allow-column-edit="false"
-                    :show-selection="false"
-                    :show-settings="false"
-                    :allow-view="acl.can('integration.viewer')"
-                    :is-loading="isLoading"
+                {% block sw_integration_list_delete_modal %}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig]: Be careful with the blocks here

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig [hunk 9]
```diff
@@ -209,137 +205,159 @@
...
-                            >
-                                {{ role.name }}
-                            </sw-label>
-                        </span>
+                        <mt-button
+                            size="small"
+                            variant="critical"
+                            @click="onConfirmDelete(deleteIntegration.id)"
+                        >
+                            {{ $t('sw-integration.detail.buttonDelete') }}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig]: Depending on what's on the button, the could maybe be replaced with `global.default.delete`

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.spec.js [hunk 10]
```diff
@@ -274,4 +296,118 @@ describe('module/sw-integration/page/sw-integration-list', () => {
...
+        expect(dataTable.classes()).toContain('sw-integration-list__overview-table--hide-pagination');
+    });
+
+    it('should show integration table pagination when more pages are available', async () => {
+        const wrapper = await createWrapper();
+
+        wrapper.vm.integrations = Object.assign([...wrapper.vm.integrations], {
+            total: wrapper.vm.limit + 1,
+        });
+        await wrapper.vm.$nextTick();
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.spec.js]: Try to avoid `nextTick` in general and use `flushPromises` instead

File: src/Administration/Resources/app/administration/src/module/sw-product-stream/page/sw-product-stream-list/sw-product-stream-list.html.twig [hunk 11]
```diff
@@ -65,12 +65,28 @@
             <mt-empty-state
                 v-if="!isLoading && !total"
                 :centered="true"
-                :icon="$route.meta.$module.icon"
+                icon="solid-products"
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-product-stream/page/sw-product-stream-list/sw-product-stream-list.html.twig]: Before, the module item was used. Is it on purpose that you now use different icons? If not, I suggest to change this in the module's index.js file

File: src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-properties/sw-product-properties.html.twig [hunk 12]
```diff
@@ -155,11 +155,7 @@
                         class="sw-product-properties__empty-state has--no-search-result"
                         :icon="$route.meta.$module.icon"
                         :headline="$t('sw-product.properties.titleEmptyStateList')"
-                    >
-                        <!-- @deprecated tag:v6.8.0 - block will be removed -->
-                        {% block sw_product_properties_filled_state_body_empty_state_image %}
-                        {% endblock %}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-properties/sw-product-properties.html.twig:161]: Don't remove.

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de.json [hunk 13]
```diff
@@ -496,9 +496,10 @@
       "deleteConfirmText": "Möchtest Du die Cross-Selling-Konfiguration wirklich löschen?",
       "deleteButtonDelete": "Löschen",
       "linkOpenStreamPreview": "Vorschau öffnen",
-      "emptyStateDescription": "Für dieses Produkt existiert noch keine Cross-Selling-Konfiguration.",
-      "inheritedEmptyStateDescription": "Dieses Produkt hat vererbte Cross Sellings vom Hauptprodukt.",
-      "notInheritedEmptyStateDescription": "Du kannst jetzt benutzerdefinierte Cross-Selling-Konfigurationen für diese Variante verwenden.",
+      "emptyStateTitle": "Noch kein Cross-Selling vorhanden",
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de.json]:
```suggestion
      "emptyStateTitle": "Noch kein Cross Selling vorhanden",
```
No need for a dash here, since we have a term completely in english

File: src/Administration/Resources/app/administration/src/module/sw-settings-currency/snippet/de.json [hunk 14]
```diff
@@ -51,8 +51,12 @@
         "totalIntervalColumn": "Summe Intervall",
         "totalNetRoundingColumn": "Summe Nettokunden"
       },
-      "emptyCountryRoundings": "Hinzufügen von Nachkommastellen und Rundungsfaktoren für verschiedene Länder.",
-      "emptyCountryRoundingsNewCurrency": "Pflege die Basisinformationen, speichere die Währung und füge dann Länderspezifische Einstellungen hinzu.",
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-settings-currency/snippet/de.json:51]: Does it make sense to remove the old ones or can we re-purpose snippet keys? In terms of translations it's always more convenient to not rename snippet keys, but it's okay if it's mandatory

File: src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/component/sw-custom-field-list/sw-custom-field-list.spec.js [hunk 15]
```diff
@@ -215,6 +277,7 @@ describe('src/module/sw-settings-custom-field/component/sw-custom-field-list/sw-
 
         await wrapper.vm.onSaveCustomField(newCustomField);
         await flushPromises();
+        await flushPromises();
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/component/sw-custom-field-list/sw-custom-field-list.spec.js]: There should never be a need to use flushPromises twice, I'd try to solve this withouth. Same with nextTick. This shouldn't be comboed with flushPromises either

File: src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/page/sw-settings-custom-field-set-create/sw-settings-custom-field-set-create.html.twig [hunk 16]
```diff
@@ -1,15 +1,19 @@
 {% block sw_settings_custom_field_set_detail_content_detail_custom_field_list %}
 <mt-card
-    class="sw-settings-set-create__option-list-empty-state__wrapper"
+    class="sw-settings-custom-field-set-create__custom-fields-card sw-settings-set-create__option-list-empty-state__wrapper"
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/page/sw-settings-custom-field-set-create/sw-settings-custom-field-set-create.html.twig]: BEM does not have selectors with 2 underscores in one selector. Please adjust accordingly

File: src/Administration/Resources/app/administration/src/module/sw-settings-tag/page/sw-settings-tag-list/sw-settings-tag-list.scss [hunk 17]
```diff
@@ -49,6 +49,11 @@
         width: 100%;
         height: 100%;
 
+        &--is-empty {
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-settings-tag/page/sw-settings-tag-list/sw-settings-tag-list.scss]: Please try `&.is--empty` instead

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig [hunk 18]
```diff
@@ -6,34 +6,36 @@
...
-                size="small"
-                variant="form"
-                @search-term-change="onSearch"
-            />
+                <sw-simple-search-field
+                    v-model:value="term"
+                    size="small"
+                    variant="form"
+                    @search-term-change="onSearch"
+                />
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig]: That wouldn't pass ESLint. We don't indent due to blocks

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig [hunk 19]
```diff
@@ -45,118 +47,123 @@
...
-                    >
-                        {{ $t('sw-users-permissions.roles.role-grid.labelButtonDelete') }}
-                    </mt-button>
+                        <mt-button
+                            class="sw-users-permissions-role-listing__confirm-delete-button"
+                            variant="critical"
+                            size="small"
+                            @click="onConfirmDelete(item)"
+                        >
+                            {{ $t('sw-users-permissions.roles.role-grid.labelButtonDelete') }}
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig]: Maybe replaceable with `global.default.delete`

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/index.js [hunk 20]
```diff
@@ -128,6 +128,22 @@ export default {
                 },
             ];
         },
+
+        showListingResults() {
+            return this.user && this.user.length > 0;
```
Comment (marcelbrode) [inline src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/index.js]:
```suggestion
            return this.user?.length > 0;
```

File: changelog/release-6-7-4-2/2026-05-18-migrate-administration-empty-states-to-meteor.md [hunk 21]
```diff
@@ -0,0 +1,7 @@
+---
+title: Migrate Administration empty states to Meteor
```
Comment (marcelbrode) [inline changelog/release-6-7-4-2/2026-05-18-migrate-administration-empty-states-to-meteor.md]: Needs to be deleted

**Review by marcelbrode — CHANGES_REQUESTED** [review]

File: src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-layout-assignment-modal/index.ts (hunk 3 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-layout-assignment-modal/index.ts:148]: Done: this is kept as a deprecated API surface with `@deprecated tag:v6.8.0` and marked for removal without replacement instead of being removed in this minor.

File: src/Administration/Resources/app/administration/src/module/sw-customer/view/sw-customer-detail-order/sw-customer-detail-order.html.twig (hunk 4 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-customer/view/sw-customer-detail-order/sw-customer-detail-order.html.twig:98]: Done: the old block is preserved as a deprecated empty block with `@deprecated tag:v6.8.0`, and the new Meteor empty-state markup uses the replacement block around the full empty state.

File: src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-ratings/sw-extension-ratings-card/sw-extension-ratings-card.html.twig (hunk 5 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-ratings/sw-extension-ratings-card/sw-extension-ratings-card.html.twig:41]: Done: the previous empty-state content block is kept as a deprecated block with `@deprecated tag:v6.8.0`, and the replacement block wraps the Meteor empty state.

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig (hunk 8 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig]: Handled carefully in the current branch: the old delete-modal block names are preserved as deprecated empty blocks, and the new Meteor modal sections have replacement blocks (`sw_integration_list_delete_modal`, `..._confirmtext`, `..._footer`).

File: src/Administration/Resources/app/administration/src/module/sw-settings-currency/snippet/de.json (hunk 14 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-settings-currency/snippet/de.json:51]: I kept the existing description keys (`emptyCountryRoundings` and `emptyCountryRoundingsNewCurrency`) and only added separate title/search keys because the Meteor empty state needs separate headline and search-result copy. So the old translation keys are still reused where their meaning fits.

File: src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/component/sw-custom-field-list/sw-custom-field-list.spec.js (hunk 15 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/component/sw-custom-field-list/sw-custom-field-list.spec.js]: Cleaned up in `a07d197dd7`: the test now only waits with `flushPromises()` after `onDeleteCustomField()` and no longer combines `$nextTick()` with another flush. The focused spec passes locally.

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig (hunk 18 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig]: Adjusted in the current branch: the template uses the regular indentation style rather than indenting because of Twig blocks. I also ran ESLint on the touched Admin files locally.

File: src/Administration/Resources/app/administration/src/app/component/base/sw-property-search/sw-property-search.html.twig (hunk 1 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/app/component/base/sw-property-search/sw-property-search.html.twig:226]: Restored: the block is kept as a deprecated extension point with `@deprecated tag:v6.8.0` instead of being removed in this minor.

File: src/Administration/Resources/app/administration/src/app/component/utils/sw-notification-center/index.js (hunk 2 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/app/component/utils/sw-notification-center/index.js]: Adjusted in the current branch; the unclear `a`/`b` naming is no longer present around this notification state.

File: src/Administration/Resources/app/administration/src/module/sw-flow/view/listing/sw-flow-list-flow-templates/sw-flow-list-flow-templates.html.twig (hunk 6 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-flow/view/listing/sw-flow-list-flow-templates/sw-flow-list-flow-templates.html.twig]: Adjusted: the empty-state branch now uses `v-else-if="!isLoading"`, so it is not evaluated/rendered when the listing is shown.

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/index.js (hunk 7 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/index.js]: Applied: `hasIntegrations` now uses optional chaining with `this.integrations?.length > 0`.

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig (hunk 9 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.html.twig]: Applied: the delete action now uses `global.default.delete`.

File: src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.spec.js (hunk 10 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-integration/page/sw-integration-list/sw-integration-list.spec.js]: Adjusted: this spec uses `flushPromises()` for the async update path and no longer relies on `nextTick()` here.

File: src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-properties/sw-product-properties.html.twig (hunk 12 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-properties/sw-product-properties.html.twig:161]: Restored: the image block is kept as a deprecated block with `@deprecated tag:v6.8.0` instead of being removed.

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de.json (hunk 13 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-product/snippet/de.json]: Applied: the German title now uses `Cross Selling` without the dash.

File: src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/page/sw-settings-custom-field-set-create/sw-settings-custom-field-set-create.html.twig (hunk 16 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/page/sw-settings-custom-field-set-create/sw-settings-custom-field-set-create.html.twig]: Adjusted: the class names no longer chain two BEM element separators in one selector.

File: src/Administration/Resources/app/administration/src/module/sw-settings-tag/page/sw-settings-tag-list/sw-settings-tag-list.scss (hunk 17 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-settings-tag/page/sw-settings-tag-list/sw-settings-tag-list.scss]: Applied: the SCSS now uses `&.is--empty` for this state.

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig (hunk 19 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-role-listing/sw-users-permissions-role-listing.html.twig]: Applied: the delete label now uses `global.default.delete`.

File: src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/index.js (hunk 20 above)
Comment (Weltraumakustik) [reply src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/index.js]: Applied: this now uses `this.user?.length > 0`.

File: changelog/release-6-7-4-2/2026-05-18-migrate-administration-empty-states-to-meteor.md (hunk 21 above)
Comment (Weltraumakustik) [reply changelog/release-6-7-4-2/2026-05-18-migrate-administration-empty-states-to-meteor.md]: Deleted: the changelog file is no longer part of the branch.

**Comment by fabianhueske** [comment]
Some have being implemented on our PRs. For the most prominent still missing I'll do follow-up PRs later. Should we close this? @Weltraumakustik @larskemper
